more_examples/autosnapshotting/integration_test/app_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:autosnapshotting/main.dart' as app;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+// Run:
+// flutter test integration_test/app_test.dart -d macos
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Snapshots are taken after reaching limit', (tester) async {
+    app.main();
+    await tester.pumpAndSettle();
+
+    final pageState =
+        tester.state<app.MyHomePageState>(find.byType(app.MyHomePage));
+    final theButton = find.byTooltip('Allocate more memory');
+
+    // Take first snapshot
+    const firstThreshold = app.memoryThresholdMb * 1024 * 1024;
```

Comment:
can you use the mbToBites extension method here (you'll have to mark the extension visible for testing)

Reply:
It is too minor to share. I do not want library code to expose utils for test code.

Reply:
Visible for testing is a Flutter thing and the library is Dart.
And, I do not want leak tracker to expose this API publicly.